ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -7,28 +11,41 @@ import { Animated, Platform, StyleSheet } from "react-native";
 import { getStatusBarHeight, isIphoneX } from "react-native-iphone-x-helper";
 import { NavigationScreenProps } from "react-navigation";
 import { connect } from "react-redux";
+import { ServiceId } from "../../../definitions/backend/ServiceId";
+import { ServicePublic } from "../../../definitions/backend/ServicePublic";
 import ChooserListContainer from "../../components/ChooserListContainer";
 import { withLightModalContext } from "../../components/helpers/withLightModalContext";
-import { withLoadingSpinner } from "../../components/helpers/withLoadingSpinner";
 import { ScreenContentHeader } from "../../components/screens/ScreenContentHeader";
 import TopScreenComponent from "../../components/screens/TopScreenComponent";
 import OrganizationLogo from "../../components/services/OrganizationLogo";
-import ServicesLocal from "../../components/services/ServicesLocal";
-import ServicesNational from "../../components/services/ServicesNational";
-import ServicesOther from "../../components/services/ServicesOther";
+import ServicesSections from "../../components/services/ServicesSectionsList";
```

Comment:
Repetition. Please remove one of these two

Reply:
removed within 3e8dd55e69564af5309be09820d64059dd82cc3c